### PR TITLE
maipo: Delete lua interpreters

### PR DESCRIPTION
**Pull request by cgwalters**
Closes: https://github.com/openshift/os/issues/275

### CODE REVIEW

File: host-maipo.yaml [hunk 1]
```diff
@@ -35,3 +35,8 @@ packages:
  - ceph-common glusterfs-fuse
  # OpenShift
  - origin-node origin-hyperkube origin-clients
+
+remove-from-packages:
+  # https://github.com/openshift/os/issues/275
+  - - lua
```
Comment (ashcrow) [inline host-maipo.yaml]: This doesn't look like valid YAML syntax. Just verifying that it's proper for this implementation of the parser.

Comment (cgwalters) [reply host-maipo.yaml]: http://yaml-online-parser.appspot.com/?yaml=remove-from-packages%3A%0A++%23+https%3A%2F%2Fgithub.com%2Fopenshift%2Fos%2Fissues%2F275%0A++-+-+lua%0A++++-+%2Fusr%2Fbin%2F.*&type=json

Comment (cgwalters) [reply host-maipo.yaml]: (Also I did test this locally)

Comment (ashcrow) [reply host-maipo.yaml]: Well alright :-)

**Comment by jlebon** [comment]
(Sidenote: we should tweak the `syntax-check` target to cover YAML files too).

**Pull request merged**